### PR TITLE
Validate package description length

### PR DESCRIPTION
**Pull request by mschnitzer**
This validates the length of package descriptions and shows an error message when it fails.

### CODE REVIEW

File: src/api/app/controllers/webui/package_controller.rb [hunk 1]
```diff
@@ -584,12 +584,14 @@ def save
     end
     @package.title = params[:title]
     @package.description = params[:description]
-    if @package.save
+    if @package.valid?
+      @package.save!
```
Comment (hennevogel) [inline src/api/app/controllers/webui/package_controller.rb]: *nitpick*: Same as
```ruby
if @package.save
  flash[:notice] = "Package data for '#{@package.name}' was saved successfully"
```

File: src/api/spec/models/package_spec.rb [hunk 2]
```diff
@@ -41,6 +41,20 @@
     login(user)
   end
 
+  describe 'validations' do
```
Comment (hennevogel) [inline src/api/spec/models/package_spec.rb]: You're overdoing it. I'm sure validations are tested in the activerecord code base :-)

Comment (mschnitzer) [reply src/api/spec/models/package_spec.rb]: I think testing validations is good to ensure that a change don't break them. We also have it on several other places.

Comment (hennevogel) [reply src/api/spec/models/package_spec.rb]: What can you change that will break `validates`? The activerecord source code?

Comment (mschnitzer) [reply src/api/spec/models/package_spec.rb]: Not validates but you can apply rules to `validates` as you know. So if someone is modifying this, he can break things accidentally. We even have model validations in our test suite. So I really don't see the point in removing this.

Comment (hennevogel) [reply src/api/spec/models/package_spec.rb]: If someone is changing this purposefully, that same someone will also change your test. You are not protecting the code base from anything, you are just making it harder to change.

**Pull request merged**